haskell/private/ghc_wrapper.sh

```diff
@@ -6,21 +6,33 @@ export PATH=${PATH:-} # otherwise GCC fails on Windows
 # support older bash versions.
 while IFS= read -r line; do compile_flags+=("$line"); done < "$1"
 
-# Detect if we are in the persistent worker mode
-if [ "$2" == "--persistent_worker" ]; then
```

Comment:
Uhm, IIUC this removes support for the persistent worker. That doesn't look right...

Reply:
Indeed, I suppressed this line when I created 2 versions of the script, but I did not keep it, when going back to only one version. This is an error.